src/main/java/seedu/algobase/model/problem/Description.java

```diff
@@ -10,6 +10,7 @@
 public class Description {
 
     public static final String MESSAGE_CONSTRAINTS = "Descriptions can take any values, and it should not be blank";
+    public static final Description DEFAULT_DESCRIPTION = new Description();
```

Comment:
Likewise, you can something like 

`public static final Description DEFAULT_DESCRIPTION = new Description("");`